test/management/data/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -ex
+
+exec 2>&1
+
+export HOME=/root
+cd
+
+yum install git -y
+
+curl -s https://storage.googleapis.com/golang/go1.11.6.linux-amd64.tar.gz | tar -C /usr/local -xz
+PATH=$PATH:/usr/local/go/bin
+
+go get -u github.com/mjudeikis/openshift-azure/cmd/proxy
```

Comment:
Gonna fix post-merge as it does not exist now in master :)

Reply:
Can we add this separately to master in a new PR and merge it first?